primefaces/src/main/java/org/primefaces/component/selectcheckboxmenu/SelectCheckboxMenuRenderer.java

```diff
@@ -318,19 +333,343 @@ protected void encodeMenuIcon(FacesContext context, SelectCheckboxMenu menu, boo
         writer.endElement("div");
     }
 
+    protected void encodePanel(FacesContext context, SelectCheckboxMenu menu, List<SelectItem> selectItems) throws IOException {
+        ResponseWriter writer = context.getResponseWriter();
+        String panelStyle = menu.getPanelStyle();
+        String panelStyleClass = SelectCheckboxMenu.PANEL_CLASS;
+        if (menu.getPanelStyleClass() != null) {
+            panelStyleClass += " " + menu.getPanelStyleClass();
+        }
+        // TODO: "RTLAware"!
```

Comment:
There is still a TODO here

Reply:
Yes, I asked @melloware if RTL support is a requirement. He said no, but I should keep the "TODO" comment.

Reply:
Yep I have no idea what RTL should do in the case of a SelectCheckBoxMenu so I said leave it as a TODO.